FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Tool/appsettings-example-v2.json

```diff
@@ -5,7 +5,8 @@
     "dicomServer": {
       "serverUrl": "",
       "apiVersion": "v1",
-      "authentication": "None"
+      "authentication": "None",
```

Comment:
just curious, what is the current configuration version? if v2, maybe we need to update the default appsetting.json in E2Etest and functionApp to v2.

Reply:
v2 is for DICOM pipeline (Even though V2 also support FHIR data source), the PaaS Analytics Connector now still uses v1 when deploying FHIR pipeline. So I think we can still use v1 in our E2E test.

And our DICOM E2E test all use v2 appsetting.

We seems haven't update configVersion for FunctionApp.